src/test/java/steps/StepsDefinition.java

```diff
@@ -0,0 +1,32 @@
+package steps;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import org.junit.Assert;
+import pages.WelcomePage;
+
+import static dictionary.Constants.LOGIN_BUTTON;
+import static dictionary.Constants.SIGN_UP_BUTTON;
+
+public class StepsDefinition {
+
+    private WelcomePage welcomePage;
+
+    @Given("I am in the Movie Swiper app")
+    public void iAmInTheMovieSwiperApp() {
+        welcomePage = new WelcomePage();
+        Assert.assertTrue(welcomePage.isOpenApp());
+    }
+
+    @And("I click on the {string} button")
```

Comment:
you can use cucumber to set the 2 options so that if yoo put other thing, cucumber warns you when you want to run it. Use some regex like : "red|blue|yellow"

Reply:
Will be investigated